lua/tabline/utils.lua

```diff
@@ -34,4 +34,67 @@ M.get_tabname = function(bufname, index)
     return vim.fn.fnamemodify(bufname, ':t')
 end
 
+M.is_val_in_table = function(val, tbl)
+    for _, value in pairs(tbl) do
+        if value == val then
+            return true
+        end
+    end
+    return false
+end
+
+M.is_focusable = function(win_id)
+    return vim.api.nvim_win_get_config(win_id).focusable
+end
+
+M.get_win_count = function(index)
+    local win_list = vim.api.nvim_tabpage_list_wins(index)
+
+    local buf_list = {}
+    local opts = config.get('show_window_count')
+
+    for _, win_id in pairs(win_list) do
+        if M.is_focusable(win_id) then
+            local bufnr = vim.api.nvim_win_get_buf(win_id)
+            local bt = vim.api.nvim_get_option_value('buftype', { buf = bufnr })
```

Comment:
`vim.bo[bufnr].buftype`

Reply:
Yeah, I was trying to wrap my head around this api, but this is so much more readable indeed.